src/screens/LoginPage/LoginPage.module.css

```diff
@@ -1,21 +1,20 @@
 #grid_wrapper {
-  display: grid;
   align-items: center;
-  gap: 4em;
-  grid-template-columns: 1fr 1fr;
 }
 
 .form_wrapper {
-  margin-top: 8em;
-  margin-left: 35em;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
```

Comment:
Absolute positioning may not be necessary here, as you are already using flexbox to position this element in the center of the page.

But if it is, please let me know.

Reply:
@DangaRanga sir, without absolute positioning this issue arises 
![issue](https://user-images.githubusercontent.com/64359826/122434393-adb03000-cfb4-11eb-80e7-2d97077c905d.PNG)